helium/src/scss/_home_page.scss

```diff
@@ -1,4 +1,4 @@
-.home-page, .academy-page {
```

Comment:
we can safely remove the .home-page now that the `academy-page` has been added (check that there is no other usage of the `home-page` class anywhere else in the styling